python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -460,7 +460,7 @@ def telemetry_callback(self, server):
 
         self.evt_inPosition.set_put(
             inPosition=bool(
-                server.telemetry.flags_pt2pt_move_complete
+                server.telemetry.flags_move_success
                 or server.telemetry.flags_slew_complete
```

Comment:
Please consider to use the `Flags_trackingSuccess` instead of `Flags_slew_complete` provided by the low-level controller as suggested in the description of DM-29832. We also discussed the details in DM-28339.

Reply:
Good catch. Thank you!